app/controllers/provider/admin/cms/visit_portal_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Provider::Admin::CMS::VisitPortalController < Provider::Admin::CMS::BaseController
+  # Encrypt the CMS token under a temporary SSO token and redirect to the Developer Portal
+  def with_token
+    cms_token = current_account.settings.cms_token!
+    expires_at = Time.now.utc.round + 1.minute
```

Comment:
If you accept my other suggestion this would be
```suggestion
    expires_at = Time.now + 1.minute
```

Reply:
You're right, but if I do this then the annoying Rubocop will never forgive me.

Reply:
Then we can use 
```suggestion
    expires_at = Time.now(in: "Z") + 1.minute
```

Although that's completely useless. What is rubocop complaining about? Timezone is ony relevant for presentation. Or if we extract some value out of it without regards to the zone. For example `#month`.